.github/workflows/build-ci.yml

```diff
@@ -350,12 +315,19 @@ jobs:
         uses: burrunan/gradle-cache-action@v1
         with:
           job-id: build-grpc-proxy
-          arguments: --scan grpc-proxy:prepareDocker
+          arguments: --scan outputVersion grpc-proxy:prepareDocker
           gradle-version: wrapper
 
+      - name: Get Deephaven Version
+        id: deephaven_version
+        run: |
+          echo "::set-output name=deephaven_version::$(cat build/version)"
```

Comment:
Interesting that some examples use `SCREAMING_SNAKE_CASE` and others just use `snake_case` for the outputs from set-output. I think I'd prefer screaming but meh doesn't really matter.